.github/workflows/integration.yml

```diff
@@ -0,0 +1,118 @@
+# Run secret-dependent integration tests only after /ok-to-test approval
+on:
+  pull_request:
+  repository_dispatch:
+    types: [ok-to-test-command]
+
+name: Integration tests
+
+jobs:
+  # Branch-based pull request
```

Comment:
are there other kinds of PRs?

Reply:
does this mean not a fork?

Reply:
From the example, it works for both forked PRs and branch-based PRs: https://github.com/imjohnbo/ok-to-test#example